api/grpc/readers/v1/readers.pb.go

```diff
@@ -0,0 +1,567 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.36.0
```

Comment:
Let's use the latest versions of both plugins and Protobuf compiler. You can install plugins via:
```bash
go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.36.6
go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.5.1
```
And compiler:
```bash
PROTOC_VERSION=30.2
PROTOC_ZIP=protoc-$PROTOC_VERSION-linux-x86_64.zip
curl -0L https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/$PROTOC_ZIP -o $PROTOC_ZIP
unzip -o $PROTOC_ZIP -d protoc3
sudo mv protoc3/bin/* /usr/local/bin/
sudo mv protoc3/include/* /usr/local/include/
rm -rf $PROTOC_ZIP protoc3
```